features/deep-linking/conversation-links.ts

```diff
@@ -0,0 +1,41 @@
+import { IXmtpConversationId, IXmtpInboxId } from "@/features/xmtp/xmtp.types"
+import { findConversationByInboxIds } from "@/features/conversation/utils/find-conversations-by-inbox-ids"
+import { useMultiInboxStore } from "@/features/authentication/multi-inbox.store"
+import { logger } from "@/utils/logger"
+
+/**
+ * Check if a conversation exists with the given inboxId
+ * @param inboxId The inbox ID to check
+ * @returns Promise with { exists, conversationId } - where conversationId is set if exists is true
+ */
+export async function checkConversationExists(inboxId: IXmtpInboxId): Promise<{ exists: boolean, conversationId?: IXmtpConversationId }> {
+  try {
+    // Get active user's inbox ID
+    const state = useMultiInboxStore.getState()
+    const activeInboxId = state.currentSender?.inboxId
+    
+    if (!activeInboxId) {
+      logger.warn("Cannot check conversation existence - no active inbox")
```

Comment:
wonder if we should just always throw instead, so that it shows in Sentry. otherwise with warnings we'll never know if we did something wrong somewhere and we get 10000 of these lol